apps/docs/pages/guides/platform/logs.mdx

```diff
@@ -108,6 +83,44 @@ The Logs tab displays logs emitted during function execution.
 </TabPanel>
 </Tabs>
 
+---
+
+## Logging Postgres Queries
+
+By default, query logs are disabled for new Supabase projects, as they can reveal metadata about the contents of your database (such as table and column names).
+
+To enable query logs, enable and configure the [pgAudit extension](https://www.pgaudit.org/):
```

Comment:
This statement is captured in the numbered list so it doesn't need to be repeated.
```suggestion
To enable query logs:
```